daemon/emer-daemon.c

```diff
@@ -910,94 +908,6 @@ handle_upload_finished (EmerDaemon *self)
 }
 
 static void
-update_timestamps (EmerDaemon *self)
-{
-  EmerDaemonPrivate *priv = emer_daemon_get_instance_private (self);
-
-  GError *error = NULL;
-  if (!emer_persistent_cache_get_boot_time_offset (priv->persistent_cache,
```

Comment:
There's a number of tests in `test-persistent-cache.c` which pass TRUE for that method. Are you sure they should all be removed as well?